src/modlunky2/ui/levels/tab.py

```diff
@@ -3079,49 +3127,19 @@ def save_level(
                             chunks=template_chunks,
                         )
                     )
-                    vanilla_setroom_type = (
-                        self.vanilla_setroom_type_for(theme, room_x, room_y)
-                        if save_format.include_vanilla_setrooms
-                        else LevelsTab.VanillaSetroomType.NONE
-                    )
-                    vf = []
-                    vb = []
-                    vs = []
-                    vm = ""
-                    if vanilla_setroom_type == LevelsTab.VanillaSetroomType.FRONT:
-                        vf = room_foreground
-                        vm = "the front layer"
-                    elif vanilla_setroom_type == LevelsTab.VanillaSetroomType.BACK:
-                        vf = room_background
-                        vm = "the back layer"
-                    elif vanilla_setroom_type == LevelsTab.VanillaSetroomType.DUAL:
-                        vf = room_foreground
-                        vm = "both layers"
-                        if dual:
-                            vb = room_background
-                            vs.append(TemplateSetting.DUAL)
-
-                    if vanilla_setroom_type != LevelsTab.VanillaSetroomType.NONE:
-                        template_chunks = [
-                            Chunk(
-                                comment=None,
-                                settings=vs,
-                                foreground=vf,
-                                background=vb,
-                            )
-                        ]
-                        comment_format = (
-                            "Auto-generated template to match {layer} of {template}."
-                        )
-                        level_templates.set_obj(
-                            LevelTemplate(
-                                name="setroom{y}-{x}".format(y=room_y, x=room_x),
-                                comment=comment_format.format(
-                                    layer=vm, template=template_name
-                                ),
-                                chunks=template_chunks,
-                            )
-                        )
+                    write_vanilla_room(room_x, room_y, room_foreground, room_background)
+
+            for room_y in range(15):
+                for room_x in range(8):
+                    if room_y < height and room_x < width:
+                        continue
+                    room_foreground = []
```

Comment:
Oh you can just 'multiply' a list by an int
  `room_foreground = `["0000000000"] * 8`

You can do the same with the string. I'm not sure it's an improvement, tho
  `room_foreground = ["0" * 10] * 8`